src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -311,9 +311,8 @@ export default {
 		 * When this component is used to send message to a breakout room we
 		 * pass an id of modal containing component to render properly.
 		 */
-		containerId: {
+		container: {
 			type: String,
-			default: null,
```

Comment:
GitHub Actions / eslint

Prop 'container' requires default value to be set

Reply:
By default we need it as 'undefined', and it works like we need that, now

@shgkme, is it necessary for Vue to assign a default value in that case?

Reply:
It's necessary for eslint, just specify default: undefined?

Reply:
If other changes will be approved, I'll add it